src/utils/createI13nNode.js

```diff
@@ -56,6 +56,7 @@ module.exports = function createI13nNode (Component, defaultProps, options) {
          */
         render: function () {
             var props = Object.assign({}, {
+                ref: 'wrapped',
```

Comment:
the strategy is good to me, we also did the same thing on some fluxible codes, 
https://github.com/yahoo/fluxible/blob/master/packages/fluxible-addons-react/connectToStores.js#L56
https://github.com/yahoo/fluxible/blob/master/packages/fluxible-addons-react/provideContext.js#L45

let's use `wrappedElement` to keep consistent.